packages/docs/static/schemas/synthql.config.json

```diff
@@ -63,6 +63,17 @@
                                             "type": "string",
                                             "nullable": true
                                         },
+                                        "items": {
```

Comment:
Honestly I think you should just make the whole column `type` an "any" schema e.g.

```ts
{
    schemaDefOverrides: {
        someTable: {
            type: {} <======= any
        }
    }
}
```